src/app/components/button/button.js

```diff
@@ -9,12 +9,11 @@ export default function Button(props) {
         }
     };
 
-
     return (
         <div className="button"> 
```

Comment:
looks like you're modifying the button component and are taking away props.color and props.size  and are putting in props.image instead. this will cause problems for people using this component in the way it initially was. can you revert these changes in this file, and make a new file for an image button instead?